performance/README.md

```diff
@@ -54,7 +54,7 @@ Ensure that influxdb and chronograf docker containers are running. Once they are
 Inorder to insert the results into the influxdb pass in the option `-o influxdb=http://localhost:8086/k6` to the `k6 run` like below:
 
 ```bash
-docker run -e 'BASE_URL=<Base_url>' -e 'PATH=<Path>' --network host -i -v $PWD:/src loadimpact/k6 -o influxdb=http://localhost:8086/k6 run /src/load.js
+docker run -e 'BASE_URL=<Base_url>' -e 'RESOURCE_PATH=<Path>' --network host -i -v $PWD:/src loadimpact/k6 -o influxdb=http://localhost:8086/k6 run /src/load.js
```

Comment:
the resource path is already set in the file and its fixed. We do not need to pass it here

Reply:
Addressed in a different branch